lib/compilers/ppci.js

```diff
@@ -27,6 +27,22 @@ const BaseCompiler = require('../base-compiler'),
     logger = require('../logger').logger;
 
 class PPCICompiler extends BaseCompiler {
+    filterUserOptions(args) {
+        return args.filter((item) => {
```

Comment:
Could you use an early return it makes it easier to read.

You can also just return that bool rather than needing an if statement . filter returns a bool. 

And to make @RabsRincon happy. === and !== for all the comparisons.

Reply:
you're right, i think i was confusing it with map()